src/js/components/Navigation/SettingsSectionFooter.jsx

```diff
@@ -90,7 +91,7 @@ class SettingsSectionFooter extends Component {
           <Link to="/more/terms"><TermsAndPrivacyText>Terms</TermsAndPrivacyText></Link>
         </OneRow>
         <OneRow centered={centered}>
-          {(isWebApp() && !inPrivateLabelMode) && (
+          {(isWebApp() && !inPrivateLabelMode && webAppConfig.ENABLE_PAY_TO_PROMOTE) && (
```

Comment:
This code below isn't related to PAY_TO_PROMOTE and doesn't need to be turned off.

Reply:
Ok!